atlasdb-cassandra-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraService.java

```diff
@@ -26,35 +26,38 @@
 public final class CassandraService {
     private static final Logger log = LoggerFactory.getLogger(CassandraService.class);
     private static final String CASSANDRA_CONFIG = "./cassandra.yaml";
+    private static final String CASSANDRA_AUTH_CONFIG = "./cassandra-auth.yaml";
 
     private static CassandraDaemon daemon;
 
     private CassandraService() {
         // do not instantiate
     }
 
-    public static void main(String[] args) {
+    public static void main(String[] args) throws InterruptedException {
         try {
-            CassandraService.start();
+            CassandraService.start(false);
```

Comment:
think this would look nicer as CassandraService.start(String config) and test suites call as CassandraService.start(CassandraService.DEFAULT_CONFIG) and CassandraService.start(CassandraService.AUTH_CONFIG)